### ISSUE TITLE
ERNIE BOT response </startCode> end tag

### ISSUE THREAD

**Issue by wsf1990** [gventuri/pandas-ai#389]
### 🚀 The feature

I use ERNIE BOT with langchain. And the bot response the code endswith `</startCode>` and raise can't found code error. I read the source code and found it just parse <endCode>.So can we replace the `</startCode>` to `<endCode>`? The whole response like this:
~~~
```python
<startCode>
import pandas as pd

data = {
    ....
}

df = pd.DataFrame(data)
print(df)
</startCode>
```
~~~

### Motivation, pitch

ERNIE bot cant response code endswith `<endCode>` but `</startCode>`

**Comment by gventuri** [comment]
@wsf1990 should have been fixed with version 0.7.1

**Comment by wsf1990** [comment]
I'm sorry, a new situation meet. The bot response with both two endtag,like this:
~~~
```python
<startCode>
import pandas as pd

df = pd.read_csv('data.csv')
print(df.head())
</startCode>
<endCode>
```
~~~
